lxc/utils.go

```diff
@@ -39,6 +39,33 @@ func runBatch(names []string, action func(name string) error) []batchResult {
 	return results
 }
 
+// Get devices from a list of profiles.
+func getProfileDevices(destRemote lxd.InstanceServer, serverSideProfiles []string) (map[string]map[string]string, error) {
+	profileDevices := make(map[string]map[string]string)
```

Comment:
Can you please move this a bit more down right before the `for` loop as this is the first time it's used. It keeps things together.